src/AccordionSection/AccordionHeader.js

```diff
@@ -6,12 +6,17 @@ import StyledAccordionHeadingText from './StyledAccordionHeadingText';
 import Icon from '../Icon';
 import VARIANTS from './variants';
 
-const AccordionHeaderIcon = ({ isOpen }) => (
-  <Icon mr="small" name="chevron" rotation={isOpen ? null : '270'} />
+const AccordionHeaderIcon = ({ isOpen, ml }) => (
+  <Icon mr="small" name="chevron" rotation={isOpen ? null : '270'} ml={ml} />
 );
```

Comment:
Should we spread `...props` here for any extra props rather than adding `ml` explicitly?